docs/getting_started.md

```diff
@@ -18,8 +18,8 @@ Run the following command to install the necessary components for the [libvirt](
 sudo dnf install -y libvirt virt-manager virt-viewer libvirt-client qemu-kvm qemu-img
 ```
 
-Download the [Red Hat Enterprise Linux 8.6 DVD ISO](https://developers.redhat.com/content-gateway/file/rhel-8.6-x86_64-dvd.iso) image for the x86_64 architecture from [Red Hat Developer](https://developers.redhat.com/products/rhel/download) site and copy the file to the `/var/lib/libvirt/images` directory.
-> Other architectures, versions or flavors of operating systems are not supported. For this setup, only use the RHEL 8.6 DVD image for x86_64 architecture.
+Download the [Red Hat Enterprise Linux 8.7 DVD ISO](https://developers.redhat.com/content-gateway/file/rhel-8.7-x86_64-dvd.iso) image for the x86_64 architecture from [Red Hat Developer](https://developers.redhat.com/products/rhel/download) site and copy the file to the `/var/lib/libvirt/images` directory.
```

Comment:
The URL to the file that I got from the site was different: https://developers.redhat.com/content-gateway/file/rhel/8.7.0/rhel-8.7-x86_64-dvd.iso 

That URL can only be accessed if you're logged in, though, and then it redirects to the CDN, so maybe we want to just link to the download page and not to the ISO?

Reply:
Fixed